src/components/space/charts/ActionsByUserChart.js

```diff
@@ -0,0 +1,112 @@
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import {
+  BarChart,
+  CartesianGrid,
+  XAxis,
+  YAxis,
+  Tooltip,
+  Bar,
+  ResponsiveContainer,
+} from 'recharts';
+import EmptyChart from './EmptyChart';
+import {
+  getActionNumByVerb,
+  filterActionsByUser,
+  findYAxisMax,
+} from '../../../utils/api';
+import { COLORS, CONTAINER_HEIGHT } from '../../../config/constants';
+import { DataContext } from '../../context/DataProvider';
+
+const ACTION_TYPES = [
+  'get',
+  'get_children',
+  'update',
+  'create',
+  'delete',
+  'copy',
+  'move',
+];
+
+const useStyles = makeStyles(() => ({
+  typography: { textAlign: 'center' },
+}));
+
+const ActionsByUserChart = () => {
+  const maxUsers = 10;
```

Comment:
Maybe factor out with a constant in `constants.js`? I would call it `ACTIONS_BY_USER_MAX_DISPLAYED_USERS = 10`